design-docs/CLI-Subscriptions.md

```diff
@@ -175,6 +175,49 @@ In the case of invalid options, print an error message (stderr) and set the exit
 code to 2. In the case of an API error response, print the response JSON and 
 set code to 1.
 
+## Request
+
+### Interface
+
+```
+planet subscriptions request [OPTIONS]
+
+Generate a subscriptions request.
+
+This command provides support for building the subscription request JSON used to create or 
+update a subscription. It outputs the subscription request.
+
+Options:
+  --name TEXT                     Subscription name. Does not need to be unique.
+                                  [required]
+  --asset-types TEXT              One or more comma-separated asset types [required]
+  --item-types TEXT               One or more comma-separated item-types [required]
+  --geometry JSON                 GeoJSON of area to deliver subscription to. Use 
```

Comment:
This is not required by the API, though I don't really get the use case of a subscription that would not have a geometry....

Reply:
If someone is only provisioned with SkySat tasking, they might make a geometryless subscription to just deliver all their fulfilled tasks? I'm not certain if that works with how our provisioning actually functions, and is certainly an edge case, but something to think about.

More generally, since most Planet users are already provisioned with a constrained geometry within which they have access to imagery, anyone who a) wants all their imagery and b) doesn't need clipping might not use geometry in their subscriptions.